src/main/java/dk/cngroup/trainings/spring/springassignment/config/WebSecurityConfig.java

```diff
@@ -17,23 +18,30 @@
 	protected void configure(HttpSecurity http) throws Exception {
 		http
 				.authorizeRequests()
```

Comment:
also have you considered using @PreAuthorize annotations in the conrollers / services? Again, maybe this can be done in the future, but it ensures that nobody who is not authorized can e.g. create animals using the service method

Reply:
I will look to see how that annotation is used.